docs/source/contributing/testing.rst

```diff
@@ -167,69 +166,59 @@ For example, to test the ``Circle`` VMobject which resides in
 ``manim/mobject/geometry.py``, add the CircleTest to
 ``test/test_geometry.py``.
 
-In ``test_geometry.py``, add:
+The name of the module is indicated by the variable __module_test__, that **must** be decalared in any graphical test file. The module name is used to store the graphical control data.
+
+.. important::
+    You will need to use the ``frames_comparison`` decorator to create a test. The test function **must** accept a 
+    parameter named ``scene`` that will be used like ``self`` in a standard ``construct`` method.
+
+Example : in ``test_geometry.py`` :
 
 .. code:: python
+  from tests.test_graphical_units.testing.frames_comparison import frames_comparison
```

Comment:
A space is needed. Also for the sake of having one complete example, all the imports + `__module_test__`.
```suggestion
.. code:: python
  
  from manim import *
  from tests.test_graphical_units.testing.frames_comparison import frames_comparison

  __module_test__ = "geometry"
  
```